app.js

```diff
@@ -0,0 +1,118 @@
+/** @format */
+
+'use strict';
+{
+  // 要素を取得
+  const btnsDiv = document.getElementById('buttons');
+  const btnStart = document.querySelector('.btnStart');
+  const head = document.querySelector('.header');
+  const question = document.querySelector('.question');
+  const genre = document.querySelector('.genre');
+  const difficult = document.querySelector('.difficult');
+
+  // 変数を定義
+  let val = 0;
+  let correctVal = 0;
+
+  // 開始ボタンのイベント
+  btnStart.addEventListener('click', () => {
+    getQuiz(val);
+  });
+
+  // 非同期処理
+  function getQuiz(val) {
+    return new Promise((resolve) => {
+      setTimeout(() => {
+        resolve(val);
+      }, 1000);
+
+      head.textContent = '取得中';
+      question.textContent = '少々お待ちください。';
+      btnsDiv.removeChild(btnStart);
+    }).then(() => {
+      callAPI(val);
+    });
+  }
```

Comment:
`Promise` でも問題はないのですが…せっかく `async` `await` を使うならこのようにするとよりよいかと思いました。
```suggestion
  async function getQuiz(val) {
    head.textContent = '取得中';
    question.textContent = '少々お待ちください。';
    btnsDiv.removeChild(btnStart);
    await callAPI(val);
  }
```